lib/tasks/eu_exit_business_finder.rake

```diff
@@ -0,0 +1,134 @@
+namespace :eu_exit_business_finder do
+  desc <<-DESC
+    Updates all documents that are tagged with a facet_value with a replacement value
```

Comment:
```suggestion
    Re-tags all documents tagged with a deprecated facet_value with its replacement facet_value
```

Reply:
I've reworked this rake task, so the comment is no longer present